docs/pages/try-out-teleport/docker-compose.mdx

```diff
@@ -57,11 +57,13 @@ $ docker-compose -f teleport-lab.yml up -d
 ```
 
 <Admonition type="note" title="Note">
+
```

Comment:
This extra white space is not common in our docs when we use Admonitions. I'd recommend removing it to be consistent with other uses.